app/views/contexts/section_summary_context.py

```diff
@@ -62,7 +62,7 @@ def __call__(self, return_to: Optional[str] = "section-summary") -> Mapping:
         }
 
     @cached_property
-    def section(self):
+    def section(self) -> Any:
         return self._schema.get_section(self.current_location.section_id)
 
```

Comment:
Should be explicitly typed.
```suggestion
    def section(self) -> ImmutableDict:
        # Type ignore: The section has to exist at this point
        section: ImmutableDict = self._schema.get_section(self.current_location.section_id) # type: ignore
        return section
```

Reply:
ok